docs/examples/Station.ipynb

```diff
@@ -40,22 +40,22 @@
    "source": [
     "## What is a Station\n",
     "\n",
-    "Experimental setups are large, comprising of many instruments; and instruments tend to be quite complex in that they comprise many parameters and other stateful parts. It deems useful to have a bucket where all of them can be conveniently stored, and accessed.\n",
+    "Experimental setups are generally large as they consist of many instruments. Each instrument in a given setup tends to be quite complex due to the fact that each comprises a variety of adjustable parameters and other stateful parts. As a result, it deems useful to have a bucket in which the necessary information concerning the instruments that are going to be used in a particular experiment can be conveniently stored, and accessed.\n",
     "\n",
-    "Here is where the concept of station comes into play. Instruments, parameters, and other \"components\" can be added to a station. As a result, the user gets an station instance that can be referred to in order to access those \"components\".\n",
+    "The concept of a station, in essence, is a programmatical representation of such a bucket. Instruments, parameters, and other \"components\" of an experiment can be added to a station. The user gets a station instance that can be referred to in order to access those \"components\".\n",
     "\n",
-    "Moreover, stations are very helpful when capturing the state of the experimental setup, known as snapshot. Refer to the respective section about station snapshot below.\n",
+    "A station can be configured from a text file which simplifies the initialization of the instruments. In particular, in this tutorial, we shall provide an example configuration of a station via a YAML file.\n",
     "\n",
-    "Last but not least, station can be configured in a text file which simplifies initialization of the instruments. Read more of this below."
+    "A special use case of a station in an experiment would be the capturing the state of an experimental setup, known as a snapshot. We shall devote a subsection for the concept of a snapshot as a particular method of the station. "
```

Comment:
the "as a particular method of the station" is confusing. what does it suppose to mean?
```suggestion
    "A special use case of a station in an experiment would be the capturing of the state of an experimental setup, known as a snapshot. We shall devote a subsection for the concept of a snapshot. "
```

Reply:
I wanted to point out that snapshot is in fact a more general concept, while, here we deal with it as a method of the station object. But we can drop that line as you suggested, no problem.